config/git_fetcher.go

```diff
@@ -190,4 +190,5 @@ var safeKeys = []string{
 	"lfs.locksverify",
 	"lfs.pushurl",
 	"lfs.url",
+	"lfs.skipdownloaderrors",
```

Comment:
One suggestion to keep these in alphabetical order:

```suggestion
	"lfs.skipdownloaderrors",
	"lfs.url",
```